core/platform/memcache/gae_memcache_services_test.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the gae_memcache_services."""
+
+from core.platform.memcache import gae_memcache_services
+from core.tests import test_utils
+
+
+def internet_on():
```

Comment:
I don't think we need these kinda testing, we can leave this situation as-is because it looks like a functionality, not as a testing.
We can leave the part of testing which depends on internet connectivity.
@seanlip, can you correct me on this?